### PR TITLE
First endpoint implementation for ACM

### PR DESCRIPTION
**Pull request by tisnik**
# Description

First endpoint implementation for ACM

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

N/A

### CODE REVIEW

**Review by quarckster — APPROVED** [review]

File: server/endpoints.go [hunk 1]
```diff
@@ -31,6 +31,9 @@ const (
 	OldReportEndpoint = "report/{org_id}/{cluster}"
 	// ReportEndpoint returns report for provided {cluster}
 	ReportEndpoint = "clusters/{cluster}/report"
+	// ReportForListOfClustersEndpoint returns rule returns reports for provided list of clusters
```
Comment (Bee-lee) [inline server/endpoints.go:34]: minor typo, otherwise LGTM

**Review by Bee-lee — APPROVED** [review]

File: server/handlers_test.go [hunk 2]
```diff
@@ -69,6 +70,7 @@ func TestHTTPServer_ReportEndpoint(t *testing.T) {
 }
 
 func TestHTTPServer_ReportEndpoint_WithOnlyOSDEndpoint(t *testing.T) {
+	time.Sleep(1 * time.Second)
```
Comment (Sergey1011010) [inline server/handlers_test.go:73]: that's strange

**Review by Sergey1011010 — APPROVED** [review]

File: server/endpoints.go (hunk 1 above)
Comment (tisnik) [reply server/endpoints.go:34]: ah my bad!

File: server/handlers_test.go (hunk 2 above)
Comment (tisnik) [reply server/handlers_test.go:73]: it is, but it lowers the chance to get unit test failure. We have to use different approach though (this does not fix DL, of course)

**Pull request merged**